pims/api.py

```diff
@@ -172,3 +164,12 @@ def _recursive_subclasses(cls):
     # Source: http://stackoverflow.com/a/3862957/1221924
     return (cls.__subclasses__() +
         [g for s in cls.__subclasses__() for g in _recursive_subclasses(s)])
+
+def _drop_dot(s):
+    if len(s) == 0 or s[0] != '.':
```

Comment:
There is also `s.startswith('.')` for readability and probably a minor performance advantage.